client/go/Taskfile.yml

```diff
@@ -17,9 +17,9 @@ version: '3'
 vars:
   # This needs to be updated if the file moves.
   REPO_ROOT: '{{joinPath .TASKFILE_DIR "../.."}}'
-  # TODO: Output to $REPO_ROOT/output/client/tun2socks instead.
-  OUT_DIR: '{{joinPath .REPO_ROOT "client/output/build"}}'
-  BIN_DIR: '{{joinPath .OUT_DIR "/bin"}}'
+  OUTPUT_DIR: '{{joinPath .REPO_ROOT "output"}}'
```

Comment:
There's OUTPUT_DIR and OUT_DIR. That's confusing.

Perhaps call this OUTPUT_ROOT.